src/DigitalTwin/CodeGeneratorCore.ts

```diff
@@ -696,7 +689,22 @@ export class CodeGeneratorCore {
     let result = false;
     await vscode.window.withProgress(
         {location: vscode.ProgressLocation.Notification, title: processTitle},
-        async () => {
+        async (progress) => {
+          progress.report({increment: 0});
+
+          setTimeout(() => {
+            progress.report({increment: 10, message: `still going...`});
+          }, 1000);
+
+          setTimeout(() => {

```

Comment:
why need to report 40%?

Reply:
Show the progress of the download pregress of the CodeGen CLI, Similar to [https://github.com/microsoft/vscode-extension-samples/tree/master/progress-sample](https://github.com/microsoft/vscode-extension-samples/tree/master/progress-sample)

---
In reply to: [361902619](https://github.com/microsoft/vscode-iot-workbench/pull/894#discussion_r361902619) [](ancestors = 361902619)